### PR TITLE
[wasm][debugger][tests] Add tests for multiple breakpoints inline

### PR DESCRIPTION
**Pull request by tqiu8**
Add 2 tests for checking multiple inline breakpoints

### CODE REVIEW

File: sdks/wasm/DebuggerTestSuite/Tests.cs [hunk 1]
```diff
@@ -123,6 +123,70 @@ public class SourceList : DebuggerTestBase {
...
+			await insp.Ready (async (cli, token) => {
+				ctx = new DebugTestContext (cli, insp, token, scripts);
+
+				var bp1_res = await SetBreakpoint ("dotnet://debugger-test.dll/debugger-array-test.cs", 197, 44);
+				Assert.EndsWith ("debugger-array-test.cs", bp1_res.Value["breakpointId"].ToString());
+				Assert.Equal (1, bp1_res.Value ["locations"]?.Value<JArray> ()?.Count);
+
+				var loc = bp1_res.Value ["locations"]?.Value<JArray> ()[0];
+
+				Assert.NotNull (loc ["scriptId"]);
```
Comment (radical) [inline sdks/wasm/DebuggerTestSuite/Tests.cs:142]: You could use `CheckLocation ("dotnet://debugger-test.dll/debugger-array-test.cs", 197, 44)`.

File: sdks/wasm/DebuggerTestSuite/Tests.cs [hunk 2]
```diff
@@ -123,6 +123,70 @@ public class SourceList : DebuggerTestBase {
 			});
 		}
 
+		[Fact]
+		public async Task CheckBreakpoint () {
```
Comment (radical) [inline sdks/wasm/DebuggerTestSuite/Tests.cs:127]: The test name should be a little descriptive.

File: sdks/wasm/DebuggerTestSuite/Tests.cs [hunk 3]
```diff
@@ -123,6 +123,70 @@ public class SourceList : DebuggerTestBase {
...
+		public async Task Create0Breakpoint () {
+			var insp = new Inspector ();
+
+			var scripts = SubscribeToScripts(insp);
+
+			await Ready ();
+			await insp.Ready (async (cli, token) => {
+				ctx = new DebugTestContext (cli, insp, token, scripts);
+
+				var bp1_res = await SetBreakpoint ("dotnet://debugger-test.dll/debugger-array-test.cs", 197, 0);
```
Comment (radical) [inline sdks/wasm/DebuggerTestSuite/Tests.cs:169]: The column number here is the only difference between the tests, correct? You can use `[InlineData (..)]` for that. See other uses of `InlineData` in the tests.

Comment (radical) [reply sdks/wasm/DebuggerTestSuite/Tests.cs:169]: Also, you might want to add another test that has multiple possible breakpoints on the same line, so when you use `col==0`, you get multiple breakpoints set.

Comment (radical) [reply sdks/wasm/DebuggerTestSuite/Tests.cs:169]: Ignore that last comment!

File: sdks/wasm/DebuggerTestSuite/Tests.cs [hunk 4]
```diff
@@ -123,6 +123,65 @@ public class SourceList : DebuggerTestBase {
 			});
 		}
 
+		[Fact]
+		public async Task CheckMultipleBreakpoints () {
```
Comment (radical) [inline sdks/wasm/DebuggerTestSuite/Tests.cs:127]: Maybe `CheckMultipleBreakpointsOnTheSameLine`?

File: sdks/wasm/DebuggerTestSuite/Tests.cs [hunk 5]
```diff
@@ -123,6 +123,65 @@ public class SourceList : DebuggerTestBase {
...
+				Assert.Equal (1, bp2_res.Value ["locations"]?.Value<JArray> ()?.Count);
+
+				var loc2 = bp2_res.Value ["locations"]?.Value<JArray> ()[0];
+
+				CheckLocation ("dotnet://debugger-test.dll/debugger-array-test.cs", 197, 49, scripts, loc2);
+			});
+		}
+
+		[Fact]
+		public async Task Create0Breakpoint () {
```
Comment (lewing) [inline sdks/wasm/DebuggerTestSuite/Tests.cs:155]: either make this work with Theory and multiple data or make the name match the other test but with a 0 somewhere in the name ;)

**Comment by lewing** [comment]
Test only changes, no impact on the release

**Comment by lewing** [comment]
@monojenkins backport master

**Pull request merged**